build.proj

```diff
@@ -13,19 +13,7 @@
     <Exec Command="dotnet build &quot;$(MSBuildThisFileDirectory)src\coverlet.collector\coverlet.collector.csproj&quot; -c $(Configuration)" />
   </Target>
 
-  <Target Name="PublishMSBuildTaskProject" AfterTargets="BuildAllProjects">
```

Comment:
No sure about that, @tonerdo we never talked about, why you build all in a `build` directory?For possible attach Debug?
With this update under build we'll have only generated packages when we build repo in release.

Reply:
@AArnott if I understood well we can pass `CoverletToolsPath` in "repro" issue(for intance) and attach local?

Reply:
Yes, that would work. But you don't *need* to do that, since the `CoverletToolsPath` property will be set to that local build path automatically when you build your tests.

Reply:
> Yes, that would work. But you don't need to do that, since the CoverletToolsPath property will be set to that local build path automatically when you build your tests.

I mean sometimes when it's not possible understand bugs using tests I attach to user repro by coping targes/props inside provided test csproj. Now I can attach and "live debug" without update csproj because using `CoverletToolsPath` I'm redirecting task load to "dev" binaries under my coverlet dev solution, right?

Reply:
Correct

Reply:
very good.